### ISSUE TITLE
API :: Connect Pull Requests To An Issue

### ISSUE THREAD

**Issue by lots0logs** [ZenHubIO/support#1251]
Is this currently possible?

**Comment by kendramoroz** [comment]
Hey @lots0logs thanks for sending this our way! On our list of enhancements to the API, but not yet available. We'll keep you posted on updates!

**Comment by kamura** [comment]
I need this API.

**Comment by nibynool** [comment]
I'd love to see this feature.

I'm currently trying to detect what PRs can be merged and to publish a notice to an internal system, but I need to filter based on tags and other details on the parent issue.  Not being able to get this relationship (either PRs for an issue, or the issue for a PR) is stopping me from this.

Any idea when this may become available, or is there anything we can do to get it happening?

**Comment by ckarawani** [comment]
this is related to issue on the API Repo:
https://github.com/ZenHubIO/API/issues/88

(@arotondo fyi)

**Comment by ckarawani** [comment]
Hey @kendramoroz has there been any movement on this issue? 
Cheers!

**Comment by kendramoroz** [comment]
Hey @ckarawani thanks for checking in! We haven't yet bumped this into in progress. Our current priorities is on shipping #139 for visibility. 

I'll ping the team on this as it seems to be surfacing a bit more these days. We'll post back if we pull this one to the top of the list!

**Comment by devzsolt** [comment]
@kendramoroz it would be great to see this endpoint for our team's automation. Could you please tell if you're already working on it or is there an ETA perhaps? Thanks a lot!

**Comment by lots0logs** [comment]
@kendramoroz ***\*bump\****

**Comment by ckarawani** [comment]
Hey @kendramoroz, any updates on this. Our team is anxiously waiting for this feature on the API

**Comment by lots0logs** [comment]
@kendramoroz If y'all would rather implement new behavior that automatically links PRs to the issues they "fix" (using the Fix/Fixes statements in the description), I'd be glad to forget all about this API enhancement :wink:

**Comment by kendramoroz** [comment]
Hey everyone—At the moment we don't have any slotted API improvements coming down the pipeline in the next quarter. You can view [our roadmap here](https://view.attach.io/H1TPiia-7) for more visibility on where time is being spent! 

We'll keep everyone updated as changes to PRs, automation, or the API come into the roadmap :)

**Comment by dkachele** [comment]
Hey @kendramoroz any chance there's been movement on this as we approach a new quarter?

**Comment by armitagemderivitec** [comment]
Hi @kendramoroz would adding this be much work? I really need it to effectively migrate from mingle which is going read-only very soon

**Comment by ckarawani** [comment]
has anyone tried just calling the `connection` API they use in the actual app? I think we might give this shot considering this request is taking such a long time... 

![image](https://user-images.githubusercontent.com/2483804/50999631-ad183300-14f8-11e9-8925-4da3cf1ee445.png)

**Comment by lots0logs** [comment]
No, please let us know how it goes if you end up trying it :pray:

**Comment by armitagemderivitec** [comment]
I tried this programatically and it worked! The only caveat was that it did not accept the same api authentication token I am using elsewhere, I had to scrape the token using chrome dev tools as above.
